src/main/api/retrieveUserByLoginIdWithLoginIdTypes.json

```diff
@@ -0,0 +1,32 @@
+{
+  "uri": "/api/user",
+  "comments": [
+    "Retrieves the user for the loginId, using specific loginIdTypes."
+  ],
+  "method": "get",
+  "methodName": "retrieveUserByLoginIdWithLoginIdTypes",
+  "successResponse": "UserResponse",
+  "errorResponse": "Errors",
+  "overload": "retrieveUserByLoginId",
```

Comment:
sister property to `"overloads": true` in the other JSON. Specifies which "base" methods this overloads.